### ISSUE TITLE
Git clone doesn't work with the samples

### ISSUE THREAD

**Issue by nojaf** [fable-compiler/fable-doc#21]
Hi , when reading https://fable.io/fable-doc/3-steps/choose-a-template.html I wondered whether you could simply download a directory using git.

```
PS C:\Temp> git clone https://github.com/fable-compiler/fable2-samples/tree/master/browser
Cloning into 'browser'...
fatal: repository 'https://github.com/fable-compiler/fable2-samples/tree/master/browser/' not found
```

That doesn't seem to work and when quickly googling it seems like alternatives can be used:
- https://stackoverflow.com/questions/7106012/download-a-single-folder-or-directory-from-a-github-repo
- https://stackoverflow.com/questions/600079/how-do-i-clone-a-subdirectory-only-of-a-git-repository/52269934#52269934

Another option might be to zip the samples and publish them as a release.
One way or another the current `git clone` instruction doesn't work.

**Comment by whitetigle** [comment]
You're right, my mistake. The link was indeed intended to lead directly to the source and not to clone the github repo. But I went too fast so I ended up mixing both.

So , maybe the easiest way for now would be to link to the repo (`git clone https://github.com/fable-compiler/fable2-samples/`) and just explain that there are subdirectories. And then, well I don't know what woudl be the easiest move for a user. 

@MangelMaxime @alfonsogarciacaro Any idea regarding this?

**Comment by MangelMaxime** [comment]
From a user perspective, the easier move is probably real templates.

We could create X templates or have 1 template with X options to choose which version to generate.

**Comment by whitetigle** [comment]
> 1 template with X options to choose which version to generate.

Any sample somewhere of this kind of template?

**Comment by MangelMaxime** [comment]
The SAFE template is doing it for example.

**Comment by whitetigle** [comment]
ok I'll check that! Thanks!

**Comment by alfonsogarciacaro** [comment]
Well, it's been a while since I haven't tried to create any dotnet template but you know my opinion: they're extremely hard to maintain, especially when they have multiple options (I guess this is the case for all templating systems).

I know this is debatable but I think templates, unless you put a lot of effort in it like in the SAFE stack template, are only good for very simple scaffolds. Also from the user perspective as this report seems to show: https://github.com/dotnet/fsharp/issues/6785

**Comment by alfonsogarciacaro** [comment]
About the docs, I will fix them by suggesting to clone the entire repo and navigate to each folder.

**Comment by alfonsogarciacaro** [comment]
Ups, that's already done 😅 Thanks!

**Comment by alfonsogarciacaro** [comment]
Fixed by #22
